source/org/zfin/antibody/Antibody.java

```diff
@@ -37,6 +38,11 @@ public class Antibody extends Marker {
     @JsonView(View.AntibodyDetailsAPI.class)
     private List<Marker> antigenGenes;
 
+    @JsonView(View.AntibodyDetailsAPI.class)
+    @JsonProperty("abregistryID")
+    @Transient
+    private String ABRegistryID;
```

Comment:
Can we use lower case for this java attribute?

Reply:
Yes, good suggestion.